src/controller/metakg.py

```diff
@@ -0,0 +1,34 @@
+import logging
+from collections.abc import Mapping
+
+from model import MetaKGDoc
+from utils.metakg.parser import Parser
+from .base import AbstractWebEntity, decoder
+
+logger = logging.getLogger(__name__)
+
+
+class MetaKGEntity(AbstractWebEntity, Mapping):
+    LOOKUP_FIELDS = []
+    MODEL_CLASS = MetaKGDoc
+
+    @classmethod
+    def create_by_smartapis(cls, smartapi_entities, include_reasoner=False):
+        for smartapi_entity in smartapi_entities:
+            extra_data = {
+                "id": smartapi_entity._id,
+                "url": smartapi_entity.url,
+            }
+            raw_data = smartapi_entity._doc._raw
+            original_data = decoder.to_dict(decoder.decompress(raw_data))
+            parser = Parser()
+            metadatas = parser.get_non_TRAPI_metadatas(original_data, extra_data)
+            if include_reasoner:
+                metadatas += parser.get_TRAPI_metadatas(original_data)
+            for metadata in metadatas:
+                cls.create_doc(metadata)
```

Comment:
possible to use bulk save here? Will it improve performance significantly?

This might be relevant: https://github.com/elastic/elasticsearch-dsl-py/issues/403

Reply:
another issue here, if we continue saving doc one at a time, is many line of elasticsearch logger output like this:

```
INFO:elasticsearch:POST http://localhost:9200/metakg_docs/_doc [status:201 request:0.049s]

```
we probably want to suppress that for cleaner log output.

Reply:
> possible to use bulk save here? Will it improve performance significantly?

Let me check first, if it is possible, I will update to use bulk save.

> another issue here, if we continue saving doc one at a time, is many line of elasticsearch logger output like this:

Sure, there logs will be suppressed.

Reply:
it's updated @newgene